### ISSUE TITLE
Mutliple "Name of" abilities used

### ISSUE THREAD

**Issue by thedukk** [doxxx/ffxiv-craft-opt-web#199]
While making a macro for Tama-Hagane Ingot on Blacksmith, the Optimizer created a macro with both Name of Earth and Name of Water being used.

**Comment by doxxx** [comment]
I’m out of touch with the game, is this a bad thing?

**Comment by doxxx** [comment]
Please provide your crafter level, craftsmanship, control and CP values.

**Comment by Ermad** [comment]
I am not sure why the optimizer is adding in multiple Name uses, but there're already checks in place so uses after the first do not provide any benefit.

Maybe need to have it add to the wastedAction count, instead of just providing no buff.

**Comment by doxxx** [comment]
Yes, I've actually just done that. I wasn't able to reproduce the original scenario though, so it's a speculative fix.

**Comment by doxxx** [comment]
@thedukk If you can, please try this again on the beta site.

**Comment by ShammyLevva** [comment]
This issue should become irrelevant in 5.0 as all "Name of" & "Brand of" abilities have gone replaced by a single "Name of the Elements" and "Brand of the Elements".

**Comment by doxxx** [comment]
When is 5.0 coming out?

**Comment by ShammyLevva** [comment]
In 8 hours 45 minutes.

**Comment by ShammyLevva** [comment]
Patch Notes https://eu.finalfantasyxiv.com/lodestone/topics/detail/26098a6854d60f4f05dd5ea152c4e85e8447a999?utm_source=launcher&utm_campaign=eu_lodestone&utm_medium=head_no2

Level cap raised to 80. New recipes, new actions. Some old actions removed. Some caps on skill-ups.  Patch notes have a lot of the details. 

Full launch is on Tues 2nd July 10am. Early Access is in 8 hrs 40 mins, no not really waiting and counting down, honest :)

**Comment by doxxx** [comment]
Yeah... gonna depend on contributions to implement new actions. :)

**Comment by ShammyLevva** [comment]
Happy to assist coding etc let me know what you need.
